httpapi/v2/attribute_api.go

```diff
@@ -0,0 +1,72 @@
+package v2
```

Comment:
Since the root of the operation is over a `document` can we add this logic to the `documents_api.go` file, or do you have any other plans around this new organization?

Reply:
I didn't want to add attribute related stuff to documents_api to keep it seperate and may potentially have other Attribute apis later